### PR TITLE
Adds Portuguese (Brazil) translation for wallet connection

### PR DESCRIPTION
**Pull request by devfelipenunes**
This pull request adds the complete Portuguese (Brazil) translation for the wallet connection module. It includes translations for connecting a wallet, sending/receiving funds, network management, and other wallet-related actions. This improves the user experience for Portuguese-speaking users.

---

## PR-Codex overview
This PR introduces the complete Portuguese (Brazil) translation for the wallet connection module, enhancing the user experience for Portuguese-speaking users by providing translations for various wallet-related actions.

### Detailed summary
- Added `"pt_BR"` to the locale types.
- Implemented Portuguese translations in various files for wallet connection, injected wallet, and smart wallet functionalities.
- Created `br.ts` files with localized strings for `ConnectLocale`, `InjectedWalletLocale`, `SmartWalletLocale`, and `InAppWalletLocale`.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by joaquim-verges** [comment]
Thank you @devfelipenunes ! there's a couple more files that contain strings to translate, could you look into translating thos too? look for all the `en.ts` files in the repo, you'll see there's different ones

File: .changeset/olive-camels-live.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"thirdweb": major
```
Comment (joaquim-verges) [inline .changeset/olive-camels-live.md:2]:
```suggestion
"thirdweb": patch
```

**Comment by devfelipenunes** [comment]
@joaquim-verges Can you help me? I'm having trouble merging.

**Comment by joaquim-verges** [comment]
@devfelipenunes merged! thanks again!

**Pull request merged**